smart-contracts/assembly/contracts/FT/token-external.ts

```diff
@@ -0,0 +1,25 @@
+import { Address, Storage } from '@massalabs/massa-as-sdk';
+import { balanceKey } from './token-internals';
+
+export function getCoinAddressCreation(
```

Comment:
I wasn't very straightforward to get the purpose of this function. Maybe we could improve the name or add a description. Suggestion: getBalanceEntryCost